justfile

```diff
@@ -16,60 +18,78 @@ linter-setup:
   source {{hermitActivationScript}} || true
   pre-commit install -f
 
-# Vanilla
-
 create-vanilla-deployment-file:
-   kustomize build {{kustomizeVanillaOverlay}} --output deploy/k8s/cloudbeat-ds.yaml
-
-build-deploy-cloudbeat: build-cloudbeat load-cloudbeat-image deploy-cloudbeat
-
-build-load-both: build-deploy-cloudbeat load-pytest-kind
-
-build-deploy-cloudbeat-debug: build-cloudbeat-debug load-cloudbeat-image deploy-cloudbeat
-
-build-replace-cloudbeat: build-binary
+  cp {{env_var('ELASTIC_PACKAGE_CA_CERT')}} {{kustomizeVanillaOverlay}}
+  kustomize build {{kustomizeVanillaOverlay}} --output deploy/k8s/cloudbeat-ds.yaml
```

Comment:
env_var('ELASTIC_PACKAGE_CA_CERT') will fail if not defined. How we can ensure that ELASTIC_PACKAGE_CA_CERT was provided before?

Reply:
When following the [README](https://github.com/elastic/cloudbeat#prerequisites), this variable should be set as part of the `eval "$(elastic-package stack shellinit)"` cmd.
Anyway, this was the idea, otherwise the `kustomize build` will fail.

Reply:
added 
```
@echo "Make sure to run 'eval \$(elastic-package stack shellinit --shell \$(basename $SHELL))'"
```
to make it more clear